pylace/src/metadata.rs

```diff
@@ -668,6 +668,19 @@ impl Codebook {
             Ok(())
         }
     }
+
+    fn like(&self, new_data: PyDataFrame) -> PyResult<Self> {
+        let index: Vec<String> = ["id", "ID", "index", "INDEX"].into_iter().map(|name|
+            new_data.0.column(name)
+        ).find(|x| x.is_ok()).ok_or_else(|| PyValueError::new_err("No index column: convention for index is ID or INDEX, in upper or lower case."))?
+        .expect("The value has already been checked")
```

Comment:
Why `expect` rather than `?` ?

Reply:
It's an optional, not Result, so I'd have to add a default error, but given the previous check, it will always work.

Reply:
Oh! I missed the `?` at the end of the previous line. That makes more sense now.